sass/govuk/elements/_components.scss

```diff
@@ -0,0 +1,24 @@
+// Box with turquoise background and white text
+// Used for transaction end pages, and Bank Holidays
+.box-highlight {
```

Comment:
Feel like the file should be called as the name of the component, `box-highlight` in this case.